packages/access/src/ownable/ownable.cairo

```diff
@@ -43,6 +48,10 @@ pub mod OwnableComponent {
         pub new_owner: ContractAddress,
     }
 
+    /// Emitted when `transfer_ownership` is called on a contract that implements `IOwnableTwoStep`.
+    /// `previous_owner` is the address of the current owner.
+    /// `new_owner` is the address of the new owner. It will be the zero address if the ownership
+    /// is being renounced.
```

Comment:
```suggestion
    /// Emitted when `transfer_ownership` is called on a contract that implements `IOwnableTwoStep`.
    /// `previous_owner` is the address of the current owner.
    /// `new_owner` is the address of the pending owner.
```

The renounce mechanism doesn't emit this event.

Reply:
Yeah, sorry I meant if the ownership transfer is canceled(you set new owner to the zero address), but wrong wording and probably does not make sense to inlcude it